### ISSUE TITLE
Dictionary for the Aramaic Bible in Plain English?

### ISSUE THREAD

**Issue by Apocalypse612** [simsrw73/BibleDict#1]
Can you add support for the Aramaic Bible in Plain English? I use this one often.

Thanks!

**Comment by simsrw73** [comment]
I'm sorry for the delay in responding. These dictionaries were produced
from books in Logos Bible Software (Logos.com). Unfortunately, they don't
have the "Aramaic Bible in Plain English" available. If you know of an open
electronic version I could download, preferably as xml, I might be able to
create one.

Regards,
Randy.
